emle/_backends/_deepmd.py

```diff
@@ -171,14 +181,13 @@ def calculate(self, atomic_numbers, xyz, forces=True):
             # To be written to qm_xyz_file.
             self._max_f_std = max_f_std
 
-        # Take averages over models and return.
-        e_mean = _np.mean(_np.array(e_list), axis=0)
-        f_mean = -_np.mean(_np.array(f_list), axis=0)
-
+        # Choose model 0 to production energy and forces.   
```

Comment:
Looks like there's some trailing whitespace here. Black should take care of that.